index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/index.css">
-</head>
-<body>
-  <h1>Quiz-App</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App start page</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header id="#starthome" class="top">
+      <h1 class="top quiz-app-heading">Quiz-App</h1>
+    </header>
+
+    <main class="app-grid__scrollarea h-100vh">
+      <section class="quiz-app-card">
+        <article class="quiz-app-card-question">
+          <h2 class="quiz-app-card__heading">Question:</h2>
+          <p class="quiz-app-card-question--text"></p>
+        </article>
+        <figure class="bookmark-container">
+          <img src="" alt="" width="" height="" class="bookmark--on" />
+        </figure>
+        <button class="quiz-app-card-answer--show"></button>
+        <button class="quiz-app-card-answer--hide"></button>
+        <p class="quiz-app-card-question--answer"></p>
+        <div class="quiz-app-card-advise-tag__container">
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+        </div>
+      </section>
+    </main>
+
+    <nav class="nav-container">
+      <ul class="nav-container__list">
+        <li class="nav-container__listitem">
+          <a href="#starthome"
+            ><img
+              src=""
+              width=""
+              height=""
+              alt=""
+              class="nav-container__home-icon"
+          /></a>
+        </li>
+        <li class="nav-container__listitem">
+          <a href="#bookmarks"
+            ><img
+              src=""
+              width=""
+              height=""
+              alt=""
+              class="nav-container__bookmark bookmark--off"
+          /></a>
+        </li>
+        <li class="nav-container__listitem">
+          <a href="#create"
+            ><img
+              src=""
+              width=""
+              height=""
+              alt=""
+              class="nav-container__create-icon"
+          /></a>
+        </li>
+        <li class="nav-container__listitem">
+          <a href="#profile"
+            ><img
+              src=""
+              width=""
+              height=""
+              alt=""
+              class="nav-container__profile-icon"
+          /></a>
+        </li>
+      </ul>
+    </nav>
+  </body>
+</html>
```

Comment:
in total really structured and good to read 👍 just the naming of the classes and few things what I would do in another way...but the good thing is there a several ways 😄 
good work

Reply:
many thanks for your detailed advises 🙏, i really appreciate that you shared your thoughts with that clear and nice descriptions 👍😃 - i can all understand - i will let the head-coach decide :-)